tests/boot/snapper_rollback.pm

```diff
@@ -19,7 +19,13 @@ sub run() {
     my ($self) = @_;
 
     # assert the we are on a ro snapshot.
-    assert_screen 'linux-login', 200;
+    # assert_screen 'linux-login', 200;
+    # workaround known issue: bsc#980337
+    if (!check_screen('linux-login', 200)) {
+        record_soft_failure 'bsc#980337';
+        send_key "ctrl-alt-f1";
+        assert_screen 'tty1-selected';
+    }
```

Comment:
Thanks for your suggestions about the improvement. I will pay attention on the points.
For "select_console", I'm not sure about it here, because I read from the lib code that login action will also be performed if one console is selected for the first time, which might be failed when system is still in booting progress.